lib/ingestor-api/runtime/tests/conftest.py

```diff
@@ -21,6 +21,7 @@ def test_environ():
     os.environ["STAC_URL"] = "https://test-stac.url"
     os.environ["DATA_ACCESS_ROLE"] = "arn:aws:iam::123456789012:role/test-role"
     os.environ["DB_SECRET_ARN"] = "testing"
+    os.environ["ROOT_PATH"] = "testing"
```

Comment:
I would usually use `monkeypatch` https://docs.pytest.org/en/stable/how-to/monkeypatch.html#monkeypatching-environment-variables to patch env variables but that's ok